### ISSUE TITLE
Place of Inspector Button after Winter '24 Release (Overlap in flow builder)

### ISSUE THREAD

**Issue by jbroekhoven-Gen25** [sorenkrabbe/Chrome-Salesforce-inspector#262]
Hello!

With the new flow builder layout in the winter '24 release the inspector button is in the way of closing the subtab.. Would it be possible to change the position of the inspector button?

![image](https://github.com/sorenkrabbe/Chrome-Salesforce-inspector/assets/118989487/471529b1-1afb-4c2d-9d5e-10e694f9a163)

**Comment by matthewctrl** [comment]
Yes, please! The overlap is unfortunate but such a pain so far in the sandbox. I guess no one in the Salesforce Flow Team are using inspector..

**Comment by tprouvot** [comment]
Hi @matthewctrl ,

You should be able to move the icon in the next release of [Salesforce Inspector Reloaded](https://chrome.google.com/webstore/detail/salesforce-inspector-relo/hpijlohoihegkfehhibggnkbjhoemldh)

Check the [feature request](https://github.com/tprouvot/Salesforce-Inspector-reloaded/issues/145) for updates
